app/scripts/views/sign_up.js

```diff
@@ -226,6 +232,22 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       return this._isMailcheckEnabledValue;
     },
 
+    _checkPasswordStrength: function () {
+      var self = this;
+      var password = this.getElementValue('.password');
+      self.passwordcheck(password, function (passwordCheckStatus) {
```

Comment:
nit: would `self.passwordcheck(password, function (passwordCheckStatus) {` ever throw an error, should the callback signature be `function (error, checkResult)` ?